geode-for-redis/src/main/java/org/apache/geode/redis/internal/data/collections/SizeableByteArrayList.java

```diff
@@ -31,6 +32,57 @@
       roundUpSize(getObjectHeaderSize() + 3 * getReferenceSize());
   private int memberOverhead;
 
+  /**
+   * @param o element to remove from the list
+   * @param count number of elements that match object o to remove from the list.
+   *        Count that is equal to 0 removes all matching elements from the list.
+   * @return list of indexes that were removed in order.
+   */
+  public List<Integer> remove(Object o, int count) {
```

Comment:
The signatures of these 3 new methods could receive `byte[]` instead of `Object`, but that's not a big deal.